<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPage</class>
 <widget class="QWidget" name="MenuPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{
	
	font: 16pt &quot;Squada One&quot;;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
	border-style: solid;
	border-color: black;
	border-width: 2px;
	border-radius: 10px;
}
QLabel 
{
	
	font: 16pt &quot;Squada One&quot;;
	border-style: solid;
	border-color: black;
	border-width: 2px;
	border-radius: 5px;
}

QLabel:focus
{
	border-color: red;
}</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: none;</string>
   </property>
   <property name="text">
    <string>VALITSE TOIMINTO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Withdraw">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>NOSTO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Saldo">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SALDO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Transacts">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TAPAHTUMAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_PayCredit">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CREDITIN MAKSU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_LogOut">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>KIRJAUDU ULOS</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
